packages/components/src/actions/ActionList/index.tsx

```
@@ -32,6 +32,7 @@ import { Trigger } from '../Trigger';
 
 import type { IIconProps, IKeyOfIcons } from '../../primitives';
 import type { IPopoverProps } from '../Popover';
+import { useDebouncedCallback } from '@onekeyhq/kit/src/hooks/useDebounce';
```

ESLint

[error] '@onekeyhq/kit/src/hooks/useDebounce' import is restricted from being used by a pattern. Please avoid using @onekeyhq/kit and @onekeyhq/kit-bg in this folder

(@typescript-eslint/no-restricted-imports)